menu.css

```diff
@@ -0,0 +1,46 @@
+.menu-wrap {
+	background-color: rgba(0, 214, 170, 0.75);
+	position: fixed;
+	text-align: center;
+	z-index: 1;
+	width: 100%;
+}
+
+nav {
+	position: absolute;
+	text-align: left;
+	top: 100%;
+	left: 0;
+	background-color: rgba(0, 214, 170, 0.75);
+	width: 100%;
+	transform-origin: top;
+	transition: transform 0.4s ease-in-out;
+}
+
+.toggler {
+	visibility: hidden;
+}
+
+nav > ul {
+	margin: 0;
+	padding: 0;
+	display: flex;
+	justify-content: flex-start;
+}
+
+
+nav > ul > li {
+	list-style: none;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/burger-one/pullRequest?prid=6031618)